src/lib.rs

```diff
@@ -133,6 +133,10 @@
 //! * by a client or individual node, or
 //! * by a sufficient number of members of a group authority.
 //!
+//! # Sequence diagrams
+//!
+//! [Bootstrapping](https://cacoo.com/diagrams/th3ie0I5yOM5ymJq-F56A2.png)
```

Comment:
I feel this should be live (poss not implemented version) Plus a link to a png or similar in the repo itself (img dir) that reflects the code at this point? The png will be updated with tags update and can then be referenced in the code doc further in the codebase.